http-compiler-plugin/src/main/java/io/ballerina/stdlib/http/compiler/Constants.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package io.ballerina.stdlib.http.compiler;
+
+/**
+ * Constants related to compiler plugin implementation.
+ */
+public class Constants {
+
+    public static final String HTTP_101 = "HTTP_101";
+    public static final String HTTP_102 = "HTTP_102";
+    public static final String HTTP_103 = "HTTP_103";
+    public static final String HTTP_104 = "HTTP_104";
+    public static final String HTTP_105 = "HTTP_105";
+    public static final String HTTP_106 = "HTTP_106";
+    public static final String HTTP_107 = "HTTP_107";
+    public static final String HTTP_108 = "HTTP_108";
+    public static final String HTTP_109 = "HTTP_109";
+    public static final String HTTP_110 = "HTTP_110";
+    public static final String HTTP_111 = "HTTP_111";
+    public static final String HTTP_112 = "HTTP_112";
+    public static final String HTTP_113 = "HTTP_113";
```

Comment:
* For error codes isn't it better to use an enum. You could find a sample [here](https://github.com/ballerina-platform/module-ballerina-websubhub/blob/main/websubhub-compiler-plugin/src/main/java/io/ballerina/stdlib/websubhub/WebSubHubDiagnosticCodes.java).

Reply:
Fixed via 2fa76a2